src/Cake.Common/Tools/OctopusDeploy/OctopusDeployRunner.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Linq;
+using Cake.Core;
+using Cake.Core.IO;
+using Cake.Core.Utilities;
+
+namespace Cake.Common.Tools.OctopusDeploy
+{
+    /// <summary>
+    /// The OctopusDeploy runner
+    /// </summary>
+    public sealed class OctopusDeployRunner : Tool<CreateReleaseSettings>
```

Comment:
Shouldn't this runner and it's argument builder names reflect that they're for *releases* specifically?